### PR TITLE
Refactor: [OV-46007] Igloo rebrand - Alert, Hyperlink, Tooltip

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: packages/Alert/src/Alert.tsx [hunk 1]
```diff
@@ -1,178 +1,186 @@
...
+    appearance = "card",
     className,
-  );
-
-  const parentElement = React.useRef<HTMLDivElement>(null);
-  const [show, setShow] = React.useState(true);
-  const hasButton = button !== undefined;
-  const isHorizontal = appearance === 'horizontal';
-  const canBeClose = closable && !isHorizontal;
+    closable = true,
```
Comment (fraincs) [inline packages/Alert/src/Alert.tsx:137]: I think a common prop name for this is dismissible

File: packages/Alert/src/Alert.tsx [hunk 2]
```diff
@@ -1,178 +1,186 @@
...
-        ref={parentElement}
-        data-test={dataTest}
-        {...rest}
-      >
-        {icon !== null &&
+    const parentElement = React.useRef<HTMLDivElement>(null);
+    const [show, setShow] = React.useState(true);
+    const hasButton = button !== undefined;
+    const isHorizontal = appearance === "horizontal";
+    const canBeClose = closable && !isHorizontal;
```
Comment (fraincs) [inline packages/Alert/src/Alert.tsx:155]: isDismissible? that would follow the isHorizontal convention

File: packages/Alert/src/alert.scss [hunk 3]
```diff
@@ -14,34 +17,86 @@
...
   --ids-alert-text-color-body: #{tokens.$grey-600};
   --ids-alert-icon-size-small: 2.6rem;
   --ids-alert-icon-size-medium: 4.0rem;
-  --ids-alert-link-color: #{tokens.$electric-blue-500};
   --ids-alert-btn-margin-top: #{tokens.$space-3};
   --ids-alert-metadata-color: #{tokens.$grey-600};
 
   /* Announcement */
-  --ids-alert-color-announcement: #{tokens.$seaweed-500};
+  --ids-alert-border-color-announcement: #{tokens.$seaweed-500};
```
Comment (fraincs) [inline packages/Alert/src/alert.scss:31]: Can't we map to semantic colors?

File: packages/Alert/src/alert.scss [hunk 4]
```diff
@@ -14,34 +17,86 @@
...
+
+  /* Info */
+  --ids-alert-border-color-info: #{tokens.$sky-100};
+
+  /* Premium */
+  --ids-alert-border-color-premium: #{tokens.$dandelion-300};
```
Comment (fraincs) [inline packages/Alert/src/alert.scss:37]: Can't we map to semantic colors?

File: packages/Alert/src/alert.scss [hunk 5]
```diff
@@ -14,34 +17,86 @@
...
+
+  /* Success */
+  --ids-alert-border-color-success: #{tokens.$seaweed-500};
```
Comment (fraincs) [inline packages/Alert/src/alert.scss:40]: Can't we map to semantic colors?

File: packages/Alert/src/alert.scss [hunk 6]
```diff
@@ -14,34 +17,86 @@
...
+
+  /* Warning */
+  --ids-alert-border-color-warning: #{tokens.$creamsicle-200};
```
Comment (fraincs) [inline packages/Alert/src/alert.scss:43]: Can't we map to semantic colors?

File: packages/Alert/src/alert.scss [hunk 7]
```diff
@@ -14,34 +17,86 @@
   --ids-alert-box-shadow: #{tokens.$shadow-6};
   --ids-alert-background-color-light: #{tokens.$samoyed};
   --ids-alert-background-color-darker: #{tokens.$grey-100};
-  --ids-alert-border-left-width: 0.6rem;
+  --ids-alert-border-default-width: 0 0 0 0.6rem;
+  --ids-alert-border-width: 0 0 0 0.6rem;
+  --ids-alert-content-margin-top: #{tokens.$space-1};
```
Comment (fraincs) [inline packages/Alert/src/alert.scss:22]: should be using a semantic token

File: packages/Alert/src/alert.scss [hunk 8]
```diff
@@ -14,34 +17,86 @@
...
   --ids-alert-text-color-title: #{tokens.$grey-800};
   --ids-alert-text-color-body: #{tokens.$grey-600};
   --ids-alert-icon-size-small: 2.6rem;
   --ids-alert-icon-size-medium: 4.0rem;
```
Comment (fraincs) [inline packages/Alert/src/alert.scss:26]: 4rem

File: packages/Alert/src/alert.scss [hunk 9]
```diff
@@ -14,34 +17,86 @@
   --ids-alert-box-shadow: #{tokens.$shadow-6};
```
Comment (fraincs) [inline packages/Alert/src/alert.scss:17]: Hopper has 4 shadows. Where is that coming from?

File: packages/Alert/src/alert.scss [hunk 10]
```diff
@@ -14,34 +17,86 @@
...
   --ids-alert-background-color-light: #{tokens.$samoyed};
   --ids-alert-background-color-darker: #{tokens.$grey-100};
-  --ids-alert-border-left-width: 0.6rem;
+  --ids-alert-border-default-width: 0 0 0 0.6rem;
+  --ids-alert-border-width: 0 0 0 0.6rem;
+  --ids-alert-content-margin-top: #{tokens.$space-1};
   --ids-alert-text-color-title: #{tokens.$grey-800};
```
Comment (fraincs) [inline packages/Alert/src/alert.scss:23]: should be using a semantic token

File: packages/Alert/src/alert.scss [hunk 11]
```diff
@@ -14,34 +17,86 @@
...
   --ids-alert-text-color-body: #{tokens.$grey-600};
```
Comment (fraincs) [inline packages/Alert/src/alert.scss:24]: should be using a semantic token

File: packages/Hyperlink/src/hyperlink.scss [hunk 12]
```diff
@@ -10,6 +13,9 @@
   --ids-link-font-weight: #{tokens.$font-weight-medium};
   --ids-link-icon-color: currentcolor;
   --ids-link-icon-margin: #{tokens.$space-1};
+  --ids-link-icon-padding-top: .2rem;
```
Comment (fraincs) [inline packages/Hyperlink/src/hyperlink.scss:16]: 0.2rem

File: packages/Hyperlink/src/hyperlink.scss [hunk 13]
```diff
@@ -24,12 +30,39 @@
...
+[data-brand="workleap"] {
+  /* Default */
+  --ids-link-font-family: var(--hop-body-sm-medium-font-family);
+  --ids-link-font-size: var(--hop-body-md-medium-font-size);
+  --ids-link-font-size-small: var(--hop-body-sm-medium-font-size);
+  --ids-link-font-size-xsmall: var(--hop-body-xs-medium-font-size);
+  --ids-link-font-weight: var(--hop-body-sm-medium-font-weight);
+  --ids-link-icon-color: currentcolor;
+  --ids-link-icon-margin: var(--hop-space-inline-xs);
+  --ids-link-icon-padding-top: .12rem;
```
Comment (fraincs) [inline packages/Hyperlink/src/hyperlink.scss:42]: 0.12rem

File: packages/Tooltip/src/tooltip.scss [hunk 14]
```diff
@@ -9,12 +12,36 @@
...
+  --ids-tooltip-z-index: 999;
+  --ids-tooltip-arrow-size: .313rem;
+
+  /* Theme */
+  --ids-tooltip-text: var(--hop-neutral-text-strong);
+  --ids-tooltip-background: var(--hop-neutral-surface-active);
+  --ids-tooltip-shadow: var(--hop-elevation-raised);
+
+  /* Light */
+  --ids-tooltip-light-text: #{tokens.$grey-800};
```
Comment (fraincs) [inline packages/Tooltip/src/tooltip.scss:42]: I was expecting a semantic color

File: packages/Tooltip/src/tooltip.scss [hunk 15]
```diff
@@ -9,12 +12,36 @@
...
+  --ids-tooltip-light-background: #{tokens.$samoyed};
```
Comment (fraincs) [inline packages/Tooltip/src/tooltip.scss:43]: I was expecting a semantic color

File: packages/Tooltip/src/tooltip.scss [hunk 16]
```diff
@@ -9,12 +12,36 @@
...
+  --ids-tooltip-light-shadow: #{tokens.$shadow-12};
```
Comment (fraincs) [inline packages/Tooltip/src/tooltip.scss:44]: I was expecting a semantic shadow

File: packages/Alert/src/alert.scss (hunk 3 above)
Comment (vicky-comeau) [reply packages/Alert/src/alert.scss:31]: That's the original Igloo style. There are no semantic tokens.

File: packages/Alert/src/Alert.tsx (hunk 1 above)
Comment (vicky-comeau) [reply packages/Alert/src/Alert.tsx:137]: true, but this was not changed and not part of the scope

File: packages/Alert/src/Alert.tsx (hunk 2 above)
Comment (vicky-comeau) [reply packages/Alert/src/Alert.tsx:155]: same as above, it's not part of what changed

File: packages/Alert/src/alert.scss [hunk 17]
```diff
@@ -14,34 +17,86 @@
...
-  --ids-alert-link-color: #{tokens.$electric-blue-500};
   --ids-alert-btn-margin-top: #{tokens.$space-3};
   --ids-alert-metadata-color: #{tokens.$grey-600};
 
   /* Announcement */
-  --ids-alert-color-announcement: #{tokens.$seaweed-500};
+  --ids-alert-border-color-announcement: #{tokens.$seaweed-500};
+
+  /* Info */
+  --ids-alert-border-color-info: #{tokens.$sky-100};
```
[the remark this answers is not in the document]
Comment (vicky-comeau) [reply packages/Alert/src/alert.scss:34]: same as above

File: packages/Alert/src/alert.scss (hunk 4 above)
Comment (vicky-comeau) [reply packages/Alert/src/alert.scss:37]: it's the old Igloo style, eventually it'll be removed. Semantic tokens don't exist

File: packages/Alert/src/alert.scss (hunk 9 above)
Comment (vicky-comeau) [reply packages/Alert/src/alert.scss:17]: this is not hopper

**Review by fraincs — APPROVED** [review]

**Review by ofrogon — APPROVED** [review]

**Pull request merged**